Source/VMware.vSphereDSC/Tests/Unit/TestHelpers/VMware.VimAutomation.Core/VMwareVimTypes.cs

```diff
@@ -688,11 +690,128 @@ public override int GetHashCode()
         }
     }
 
+    public class HostVirtualSwitchBridge
```

Comment:
Are all those types needed for the current Resources or are they for future resources as well ?

Reply:
No, they are needed for the remaining VSS resources (teaming, bridge...).
Adding them now allowed me to check if my object definitions are ok.